src/NumericalAlgorithms/FiniteDifference/HighOrderFluxCorrection.hpp

```diff
@@ -0,0 +1,336 @@
+                   ++grid_index, ++stencil_index) {
+                if (grid_index < 0) {
+                  neighbor_index[dim] = static_cast<size_t>(
+                      static_cast<int>(number_of_ghost_cells) + grid_index);
+                  gsl::at(cell_centered_fluxes_for_stencil, stencil_index) =
+                      lower_neighbor_cell_centered_flux[flux_storage_index]
+                                                       [collapsed_index(
+                                                           neighbor_index,
+                                                           neighbor_extents)];
+                } else if (grid_index >=
+                           static_cast<int>(subcell_extents[dim])) {
+                  neighbor_index[dim] = static_cast<size_t>(
+                      grid_index - static_cast<int>(subcell_extents[dim]));
+                  gsl::at(cell_centered_fluxes_for_stencil, stencil_index) =
+                      upper_neighbor_cell_centered_flux[flux_storage_index]
+                                                       [collapsed_index(
+                                                           neighbor_index,
+                                                           neighbor_extents)];
+                } else {
+                  Index<Dim> volume_index = face_index;
+                  volume_index[dim] = static_cast<size_t>(grid_index);
+                  gsl::at(cell_centered_fluxes_for_stencil, stencil_index) =
+                      cell_centered_flux[flux_storage_index][collapsed_index(
+                          volume_index, subcell_extents)];
+                }
+              }
+
+              // Only do 4th-order correction for now...
+              if constexpr (CorrectionOrder >= 10) {
+                correction -=
+                    5.6689342403628117913e-5 *
+                    (gsl::at(cell_centered_fluxes_for_stencil,
+                             correction_width - 4) +
+                     gsl::at(cell_centered_fluxes_for_stencil,
+                             correction_width + 3) -
+                     9.8 * (gsl::at(cell_centered_fluxes_for_stencil,
+                                    correction_width - 3) +
+                            gsl::at(cell_centered_fluxes_for_stencil,
+                                    correction_width + 2)) +
+                     49.0 * (gsl::at(cell_centered_fluxes_for_stencil,
+                                     correction_width - 2) +
+                             gsl::at(cell_centered_fluxes_for_stencil,
+                                     correction_width + 1)) -
+                     245.0 * (gsl::at(cell_centered_fluxes_for_stencil,
+                                      correction_width - 1) +
+                              gsl::at(cell_centered_fluxes_for_stencil,
+                                      correction_width)) -
+                     409.6 * second_order_var_correction[storage_index]
```

Comment:
okay this doesn't match the doc, but is symmetric...

Reply:
The docs was just wrong... I've fixed it in a fixup